gulpfile.js

```diff
@@ -0,0 +1,38 @@
+const gulp = require('gulp'),
+  del = require('del'),
+  changeCase = require('change-case'),
+  replace = require('gulp-string-replace'),
+  rename = require('gulp-rename'),
+  exec = require('child_process').exec,
+  argv = require('yargs')
+    .usage('Usage: $0 [options]')
+    .command('create', 'Create a new component')
+    .example('$0 --component=example-widget --dest=src/app/')
+    .alias('d', 'dest')
+    .describe('d', 'The directory to place the component in.')
+    .alias('c', 'component')
+    .describe('c', 'The name of the component to create in snake case.')
+    .demandOption(['c', 'd'])
+    .argv;
+
+function copyToDest(srcArr) {
+  return gulp.src(srcArr)
+    .pipe(gulp.dest(function (file) {
+      return libraryDist + file.base.slice(__dirname.length); // save directly to dist
+    }));
+}
+
+gulp.task('create', function () {
```

Comment:
Is this to make it easier for you to create widgets on the dashboard? Or are you intending the user to run this? Or does it do something else entirely.